docker/validator-dev/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+set -e
+
+DIR="$( cd "$( dirname "$0" )" && pwd )"
```

Comment:
```suggestion
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```
([rationale](https://stackoverflow.com/a/35006505/47978))